lib/puppetserver/ca/puppet_config.rb

```diff
@@ -163,6 +165,17 @@ def explicitly_given_config_file_or_default_config_exists?
       def run(command)
         %x( #{command} )
       end
+
+      def parse_crl_usage(setting)
```

Comment:
this makes parsing easier on the other end though I don't know how I feel about mixing up the representation of this setting between here and Puppet proper.

Reply:
I'm fine doing this here. In the PR I put up, I'm renaming at least one setting to make it more accurate. I think we should do the right thing here internally as much as possible.